Libraries/CO.CDP.AspNetCore/CO.CDP.AspNetCore.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <ProjectReference Include="..\..\Services\CO.CDP.OrganisationInformation.Persistence\CO.CDP.OrganisationInformation.Persistence.csproj" />
```

Comment:
If you're planning to make this library dependent on all the projects with exceptions that we plan to map, it's likely to introduce circular dependencies. Remember that exceptions might be thrown by any layer (not only persistence).